benchmarks/high_volume_writes_test.go

```diff
@@ -37,7 +37,7 @@ var _ = Describe("Scenario: High Volume Writes", func() {
 			_ = logger.Undeploy(k8sClient, benchCfg.Logger)
 
 			// deploy loggers
-			err := logger.Deploy(k8sClient, benchCfg.Logger, writerCfg, benchCfg.Loki.PushURL())
+			err := logger.Deploy(k8sClient, benchCfg.Logger, writerCfg, benchCfg.Loki.PushURL(), writerCfg.Args)
```

Comment:
The `writerCfg` is already passed as 3rd argument. No need to redo this here.

Reply:
Fixed
DONE